.github/workflows/build-vcpkg.yml

```diff
@@ -21,348 +51,195 @@ on:
+    secrets: inherit
+
+  test-unit-docker-ubuntu-22_04:
+    if: ${{ contains('pull_request,push', github.event_name) }}
+    needs: build-docker-ubuntu-22_04
+    uses: ./.github/workflows/test-unit-gh_runner.yml
+    with:
+      os: ubuntu-22.04
+      asset-name: 'docker-package'
+    secrets: inherit
+
+  test-ui-docker-ubuntu-22_04:
+    if: ${{ contains('pull_request,push', github.event_name) }}
+    needs: build-docker-ubuntu-22_04
+    uses: ./.github/workflows/test-ui-gh_runner.yml
+    with:
+      os: ubuntu-22.04
+      asset-name: 'docker-package'
+    secrets: inherit
+
+  build-docker-ubuntu-20_04:
+    if: ${{ contains('schedule,push', github.event_name) }}
+    uses: ./.github/workflows/build-docker.yml
+    with:
+      os: ubuntu-20.04
+      single-package: true
+      containerized: false
+    secrets: inherit
+
+  build-docker-centos-8:
+    if: ${{ contains('schedule,push', github.event_name) }}
+    uses: ./.github/workflows/build-docker.yml
+    with:
+      os: centos-8
+      single-package: true
+      containerized: false
+    secrets: inherit
+
+  build-docker-centos-7:
+    if: ${{ contains('pull_request,push', github.event_name) }}
+    uses: ./.github/workflows/build-docker.yml
+    with:
+      os: centos-7
+      single-package: true
+      containerized: false
+      cmake-configuration-ex: "-DVCPKG_TARGET_TRIPLET=x64-centos-7-dynamic"
+    secrets: inherit
+
+  build-docker-amazonlinux:
+    if: ${{ contains('schedule,push', github.event_name) }}
+    uses: ./.github/workflows/build-docker.yml
+    with:
+      os: amazonlinux
+      single-package: true
+      containerized: false
+      cmake-configuration-ex: "-DVCPKG_TARGET_TRIPLET=x64-amazonlinux-dynamic"
+    secrets: inherit
+
+  build-gh_runner-ubuntu-22_04:
+    if: ${{ contains('pull_request,push', github.event_name) }}
+    uses: ./.github/workflows/build-gh_runner.yml
+    with:
+      os: ubuntu-22.04
+      single-package: true
+      containerized: false
+      upload-package: true
+      asset-name: 'gh_runner-package'
+    secrets: inherit
+  
+  test-smoke-gh_runner-ubuntu-22_04:
+    if: ${{ contains('pull_request,push', github.event_name) }}
+    needs: build-gh_runner-ubuntu-22_04
+    uses: ./.github/workflows/test-smoke-gh_runner.yml
+    with:
+      os: ubuntu-22.04
+      asset-name: 'gh_runner-package'
+    secrets: inherit
+
+  test-unit-gh_runner-ubuntu-22_04:
+    if: ${{ contains('pull_request,push', github.event_name) }}
+    needs: build-gh_runner-ubuntu-22_04
+    uses: ./.github/workflows/test-unit-gh_runner.yml
+    with:
+      os: ubuntu-22.04
+      asset-name: 'gh_runner-package'
+    secrets: inherit
+
+  test-ui-gh_runner-ubuntu-22_04:
+    if: ${{ contains('pull_request,push', github.event_name) }}
+    needs: build-gh_runner-ubuntu-22_04
+    uses: ./.github/workflows/test-ui-gh_runner.yml
+    with:
+      os: ubuntu-22.04
+      asset-name: 'gh_runner-package'
+    secrets: inherit
```

Comment:
This is a technically a "new" test but is failing, will need further investigation...